client/my-sites/themes/theme-showcase.jsx

```diff
@@ -468,6 +469,7 @@ class ThemeShowcase extends Component {
 
 		const tabFilters = this.getTabFilters();
 		const tiers = this.getTiers();
+		const isSearchV2 = ! isLoggedIn && config.isEnabled( 'themes/text-search-lots' );
```

Comment:
Minor/non-blocking/fine-for-follow-up.

This is fine for a quick fix, but I think it could be improved like this:

```suggestion
		const isSearchText =
			( isLoggedIn && config.isEnabled( 'themes/text-search-lits' ) ) ||
			( ! isLoggedIn && config.isEnabled( 'themes/text-search-lots' ) );
```

Of course with `themes/text-search-lits` turned off everywhere.

Reply:
Right now we don't have a `themes/text-search-lits` config key since it isn't needed (we aren't going to enable the new search on the LiTS). If you decide to add it we could use the logic I had before which is a bit simpler than your proposal:

```
const isSearchV2 = isLoggedIn
			? config.isEnabled( 'themes/search-v2-lits' )
			: config.isEnabled( 'themes/search-v2-lots' );
```